src/radical/saga/adaptors/lsf/lsfjob.py

```diff
@@ -252,8 +255,8 @@ def _lsfscript_generator(url, logger, jd, ppn, lsf_version, queue):
     alloc_flags.append('smt%d' % smt)
     lsf_bsubs += "#BSUB -alloc_flags '%s' \n" % ' '.join(alloc_flags)
 
-    env_string += "export RADICAL_SAGA_SMT=%d" % smt
     if jd.environment:
+        env_string += "export"
```

Comment:
So we don't export `RADICAL_SAGA_SMT` anymore, which is fine IMHO - but should't we export `RADICAL_SMT` instead?  Or maybe we do somewhere else in the code already?  The change to the test seems to indicate no, as the env is explicitly added to the job description?

Reply:
> So we don't export `RADICAL_SAGA_SMT` anymore

right, at some point it became a leftover

> but should't we export `RADICAL_SMT` instead?

this will be controlled by RP, so RP provides this env variable within `jd.environment` and then it is added to this export-statement

Reply:
> > but should't we export `RADICAL_SMT` instead?
> 
> this will be controlled by RP, so RP provides this env variable within `jd.environment` and then it is added to this export-statement

True - but since the SAGA layer knows about it, we could leave it to SAGA to export that env in general.  Well, that's out of scope for this PR I guess, just saying...